SPECS-EXTENDED/python-msal/python-msal.spec

```diff
@@ -0,0 +1,151 @@
+%global         srcname         msal
+%global         forgeurl        https://github.com/AzureAD/microsoft-authentication-library-for-python/
+%global         pypi_version    1.18.0b1
+%global         tag             %{pypi_version}
+%forgemeta
+
+# Most of the tests require network access, so they are disabled by default.
+%bcond_with     tests
+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Version:        1.18.0~b1
+Name:           python-%{srcname}
+Release:        2%{?dist}
+Summary:        Microsoft Authentication Library (MSAL) for Python
+
+License:        MIT
+URL:            %forgeurl
+Source0:        %forgesource
```

Comment:
Append #/%{name}-%{version}.tar.gz. Please check the other spec files for this as well.